src/pages/release-notes.md

```diff
@@ -35,3 +35,11 @@ A comprehensive, coherent and compatible set of development artefacts. Highlight
 - several improvements to the user-friendliness of KITs based on feedback received to date, making Tractus-X overall more accessible and welcoming to newcomers and veterans alike.
 
 details to be found [here](https://eclipse-tractusx.github.io/CHANGELOG)
+
+### Further Scope Clarification
+
+To maintain the integrity and reliability of the onboarding process, the signing of self-descriptions (GXDC-compliance check) will be temporarily suspended:
```

Comment:
GXDCH 

would write: "the signing of self-descriptions by the Gaia-X Digital Clearing House (GXDCH )will be temporarily suspended:"